docs/examples/data/torchvision/data.py

```diff
@@ -0,0 +1,54 @@
+"""Make sure the data is available"""
+import os
+import shutil
+import sys
+import time
+from multiprocessing import Pool
+from pathlib import Path
+
+from torchvision.datasets import INaturalist
+
+
+def link_file(src:str, dest:str):
+    Path(src).symlink_to(dest)
```

Comment:
This already receives paths:
```suggestion
def link_file(src: Path, dest: Path) -> None:
    src.symlink_to(dest)
```